boot/makebootable.go

```diff
@@ -529,6 +543,48 @@ func makeRunnableSystem(model *asserts.Model, bootWith *BootableSet, sealer *Tru
 	return nil
 }
 
+func buildOptionalKernelCommandLine(gadgetSnapOrDir string, model *asserts.Model) (string, error) {
+	sf, err := snapfile.Open(gadgetSnapOrDir)
+	if err != nil {
+		return "", fmt.Errorf("cannot open gadget snap: %v", err)
+	}
+	gadgetInfo, err := gadget.ReadInfoFromSnapFile(sf, nil)
+	if err != nil {
+		return "", fmt.Errorf("cannot read gadget data: %v", err)
+	}
+
+	defaults := gadget.SystemDefaults(gadgetInfo.Defaults)
+
+	var cmdlineOpt, cmdlineOptDang string
```

Comment:
same question about s/Opt/Append/ ?

nitpick: Dang is a bit of an horrible abbreviation because it's an informatl exclamation, I would just keep it fulled spelled out